package.json

```diff
@@ -65,23 +65,23 @@
   "devDependencies": {
     "@actions/core": "1.9.1",
     "@actions/github": "5.0.3",
-    "@babel/core": "7.21.8",
-    "@babel/generator": "7.21.5",
-    "@babel/parser": "7.21.8",
+    "@babel/core": "7.23.2",
+    "@babel/generator": "7.23.0",
+    "@babel/parser": "7.23.0",
     "@babel/plugin-proposal-class-properties": "7.18.6",
-    "@babel/plugin-proposal-decorators": "7.21.0",
+    "@babel/plugin-proposal-decorators": "7.23.2",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.18.6",
     "@babel/plugin-proposal-optional-chaining": "7.21.0",
     "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/plugin-syntax-object-rest-spread": "7.8.3",
-    "@babel/plugin-transform-runtime": "7.21.4",
-    "@babel/preset-env": "7.21.5",
-    "@babel/preset-react": "7.18.6",
-    "@babel/preset-typescript": "7.21.5",
-    "@babel/register": "7.21.0",
-    "@babel/standalone": "7.21.8",
-    "@babel/traverse": "7.21.5",
-    "@babel/types": "7.21.5",
+    "@babel/plugin-transform-runtime": "7.23.2",
+    "@babel/preset-env": "7.23.2",
+    "@babel/preset-react": "7.22.15",
+    "@babel/preset-typescript": "7.23.2",
+    "@babel/register": "7.22.15",
+    "@babel/standalone": "7.23.2",
+    "@babel/traverse": "7.23.2",
```

Comment:
this also resolves CG critical issue
<img width="1082" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/b0129d94-7e44-40fa-8723-289acead4204">